event_routing_backends/backends/events_router.py

```diff
@@ -28,7 +28,95 @@ def __init__(self, processors=None, backend_name=None):
         self.processors = processors if processors else []
         self.backend_name = backend_name
 
-    def send(self, event):
+    def configure_host(self, host, router):
+        """
+        Create host_configurations for the given host and router.
+        """
+        host['host_configurations'] = {}
+        host['host_configurations'].update({'url': router.route_url})
+        host['host_configurations'].update({'auth_scheme': router.auth_scheme})
+
+        if router.auth_scheme == RouterConfiguration.AUTH_BASIC:
+            host['host_configurations'].update({'username': router.username})
+            host['host_configurations'].update({'password': router.password})
+        elif router.auth_scheme == RouterConfiguration.AUTH_BEARER:
+            host['host_configurations'].update({'auth_key': router.auth_key})
+
+        if router.backend_name == RouterConfiguration.CALIPER_BACKEND:
+            host.update({'router_type': 'AUTH_HEADERS'})
+            if 'headers' in host:
+                host['host_configurations'].update({'headers': host['headers']})
+        elif router.backend_name == RouterConfiguration.XAPI_BACKEND:
+            host.update({'router_type': 'XAPI_LRS'})
+        else:
+            host.update({'router_type': 'INVALID_TYPE'})
+
+        return host
+
+    def prepare_to_send(self, events):
```

Comment:
This functionality was also broken out of `send()` and adapted to handle multiple events.